src/components/search/search.css

```diff
@@ -1,12 +1,40 @@
 .search {
   display: flex;
   justify-content: center;
+  margin: 24px;
 }
 
 .searchText {
   font-size: 1.5em;
+  height: 18px;
+  padding: 8px;
+  background: rgba(50, 50, 50, 0.2);
+  border: 1px solid #5998ff;
+}
+
+.searchText:focus, .searchButton:focus {
+  outline: none;
 }
 
 .searchButton {
-  font-size: 1.5em;
+  font-size: 1.3em;
+  position: relative;
+  height: 36px;
+  padding: 4px 15px;
+  left: 4px;
```

Comment:
Use `margin-left`, since we aren't going to use relative positioning

Reply:
Also try adding `line-height: 1;`.  It should cause the text to be centred vertically in the button instead of being a little too far to the top.  That's what I found while experimenting with this anyways.